src/hierarchies/lin_reg_uni_hierarchy.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef BAYESMIX_HIERARCHIES_LIN_REG_UNI_HIERARCHY_HPP_
+#define BAYESMIX_HIERARCHIES_LIN_REG_UNI_HIERARCHY_HPP_
+
+#include <google/protobuf/stubs/casts.h>
+
+#include <Eigen/Dense>
+#include <memory>
+
+#include "../../proto/cpp/hierarchy_prior.pb.h"
```

Comment:
Since we're generating the protos with cmake, the include path is now plain and simple
```
# include "hierarchy_prior.pb.h"
```